tidb-binlog/tidb-binlog-overview.md

```diff
@@ -50,15 +54,12 @@ The TiDB Binlog cluster is composed of Pump and Drainer.
 
 ## Notes
 
-* You need to use TiDB v2.0.8-binlog, v2.1.0-rc.5 or a later version. Older versions of TiDB cluster are not compatible with the cluster version of TiDB Binlog.
-
-* Drainer supports replicating binlogs to MySQL, TiDB, Kafka or local files. If you need to replicate binlogs to other Drainer unsuppored destinations, you can set Drainer to replicate the binlog to Kafka and read the data in Kafka for customized processing according to binlog consumer protocol. See [Binlog Consumer Client User Guide](/tidb-binlog/binlog-consumer-client.md).
-
-* To use TiDB Binlog for recovering incremental data, set the config `db-type` to `file` (local files in the proto buffer format). Drainer converts the binlog to data in the specified [proto buffer format](https://github.com/pingcap/tidb-binlog/blob/master/proto/binlog.proto) and writes the data to local files. In this way, you can use [Reparo](/tidb-binlog/tidb-binlog-reparo.md) to recover data incrementally.
+* TiDB Binlog is not compatible with the following features introduced in TiDB v5.0.0-rc and they cannot be used together:
```

Comment:
can we put `you can use [TiCDC](/ticdc/ticdc-overview.md) instead.`  after this sentence, and remove the same content in its clause